### PR TITLE
sanity check warnings

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
Warnings when creating model-configurations

image

### Testing
model-configuration operator, use lower/upper bound of distribution of a parameter to be the same, try to run and should get a warning prompt dialog

### CODE REVIEW

File: packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue [hunk 1]
```diff
@@ -188,7 +188,7 @@
 							:disabled="isSaveDisabled"
 							label="Run"
 							icon="pi pi-play"
-							@click="createConfiguration"
+							@click="createConfiguration(false)"
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue:191]: Why adding the `false` when it is the default value?
```suggestion
							@click="createConfiguration"
```

File: packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue [hunk 2]
```diff
@@ -636,10 +662,49 @@ const updateConfigFromModel = (inputModel: Model) => {
...
+const createConfiguration = async (force: boolean = false) => {
 	if (!model.value) return;
 
 	const state = cloneDeep(props.node.state);
+
+	sanityCheckErrors.value = [];
+	if (force !== true) {
+		const errors = runSanityCheck();
+		if (errors.length > 0) {
+			console.log(errors);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue]: Just to make it more digestible
```suggestion
      console.log("Model configuration sanity check errors: ");
			console.table(errors);
```

File: packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue [hunk 3]
```diff
@@ -263,6 +263,30 @@
...
+				<section style="max-height: 22rem; overflow-y: scroll">
+					<div v-for="(errString, idx) of sanityCheckErrors" :key="idx">
+						{{ errString }}
+					</div>
+				</section>
+			</template>
+			<template #footer>
+				<Button label="Ok" class="p-button-primary" @click="sanityCheckErrors = []" />
+				<Button
+					label="Ignore"
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue:282]: I feel the label `Ignore` to be a bit confusing by itself. Maybe it's just me.
```suggestion
					label="Ignore warnings and use configuration"
```

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue (hunk 1 above)
Comment (mwdchang) [reply packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue:191]: Because linter doesn't like it.

**Pull request merged**